tests/unit/test_sqs.py

```diff
@@ -46,3 +48,59 @@ def test_handle_string_max_receive_count_in_dead_letter_check():
     sqs_message = provider.SqsMessage(Message(), {})
     result = provider.SqsProvider()._dead_letter_check(queue, sqs_message, None)
     assert result is False
+
+
+def test_except_check_message_size():
+    message_body = "".join(("a" for _ in range(262145)))
+    result = False
+    try:
+        provider.check_message_size(message_body, MAXIMUM_MESSAGE_SIZE)
+    except provider.InvalidMessageContents:
+        result = True
+
+    assert result
+
+
+def test_noexc_check_message_size():
+    message_body = "a"
+    result = True
+    try:
+        provider.check_message_size(message_body, MAXIMUM_MESSAGE_SIZE)
+    except provider.InvalidMessageContents:
+        result = False
+
+    assert result
```

Comment:
since we are checking the negative case above, we can just run `check_message_size`. the test will fail if an exception occurs

```suggestion
    provider.check_message_size(message_body, MAXIMUM_MESSAGE_SIZE)
```